test/timezone/test_lookup.rb

```diff
@@ -3,12 +3,12 @@
 require 'timezone/lookup'
 require 'minitest/autorun'
 
-class TestLookup < ::Minitest::Test
+class TestLookup < Minitest::Test
   def test_test_config
     Timezone::Lookup.config(:test)
 
     assert_equal Timezone::Lookup::Test,
-      Timezone::Lookup.lookup.class
+                 Timezone::Lookup.lookup.class
```

Comment:
Is there a cop changing this behavior and can we revert to the previous behavior?

Reply:
This is a spacing lint.